org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/WorkspaceProvider.xtend

```diff
@@ -12,8 +12,12 @@ class WorkspaceProvider {
 		rootFS = configuration.gitFSRoot
 	}
 
-	def File getWorkspace(String userId) {
-		return new File(rootFS, userId)
+	def File getWorkspace(String userId, boolean separateUserWorkspaces) {
```

Comment:
Why is this property passed all the way instead of retrieved directly from the configuration object?